Simplified.xcodeproj/project.pbxproj

```diff
@@ -2089,7 +2089,7 @@
 				PRODUCT_NAME = "$(TARGET_NAME)";
 				PROVISIONING_PROFILE = "2e185b6c-271e-4b02-a05e-860b8c3831f6";
 				PROVISIONING_PROFILE_SPECIFIER = "SimplyE Development (Keychain Migration)";
-				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "DEBUG FEATURE_DRM_CONNECTOR";
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
```

Comment:
can you remove these changes and mention them in the list of manual steps instead? Otherwise the DRM build will be affected and build instructions will need to change there. I'd rather not add them there because we have automated the steps already for DRM build.